lambdas/update_new_area_status/function.py

```diff
@@ -0,0 +1,58 @@
+from upload_records.tasks import get_task_log
+from upload_records.aws import get_api_token
+from upload_records.errors import DatasetFailedError, DatasetPendingError
+from summary_analysis_batch.utils import slack_webhook
+
+import requests
+import json
+import boto3
+import csv
+
+
+def handler(event, context):
+    name = event["name"]
+    env = event["env"]
+    analyses = event["analyses"]
+    aoi_src = event["feature_src"]
+
+    geostore_ids = get_aoi_geostore_ids(aoi_src, env)
+    update_aoi_status(geostore_ids, env)
+
+
+def get_aoi_geostore_ids(aoi_src, env="production"):
+    s3_client = boto3.client("s3")
+
+    tsv_list = s3_client.list_objects(Bucket=NEW_AOI_BUCKET, Prefix=NEW_AOI_PREFIX)['Contents']
+    geostore_ids = []
```

Comment:
This should be a set. Geostore IDs are not unique in this file. I split geometries into 1x1 degree tiles.